app/components/request_form/request_form.js

```diff
@@ -20,4 +22,31 @@ export default class extends Controller {
 
     this.previewTarget.innerHTML = template(data);
   }
+
+  updateMembersCount(event) {
+    if (!event || event.detail.tags.length <= 0) {
```

Comment:
```suggestion
    if (!event || event.detail.tags.length === 0) {
```
or `< 1`... it could never be less than one right?
or `!!event.detail.tags.length`?

Reply:
@mattwr18: I like to test for `<= 0` instead of `== 0` so that all values that an integer could possibly have are handled. There are many situations (see below) in JavaScript where this comes in handy. Even though it’s rather unlikely in this specific example, I like to make it a habit.

I guess you’d have concerns about this making the code less clear?

```
> null <= 0
< true
```

```
> null == 0
< false 
```

```
> 'lorem ipsum'.indexOf('hello world')
< -1
```